tools/test/audio/test_utils/mkdir_check.m

```diff
@@ -29,7 +29,7 @@
 %
 
 function d = mkdir_check(d)
-if exist(d) ~= 7
+if ~exist(fullfile('.',d),'dir')
```

Comment:
Nitpick, in C code style we add one blank after commas, maybe we should in Matlab too. Though I'm not always doing it always myself.